pyproject.toml

```diff
@@ -68,3 +71,11 @@ extend-exclude = '''
   | reflex-generator
 )
 '''
+
+[tool.isort]
+line_length = 120
```

Comment:
Can we change the line length to 88 for consistency with black?

Is it possible to have line length defined in just one place, and specify the tool execution order to avoid inconsistent results?

If there is no automated solution it is probably better to add the instructions to the README file or CONTRIBUTING.md so people know how to get it right.

Reply:
Ended up going with 108 for black and removing the override in isort.